### PR TITLE
[CMS-1179] Separate D10 upstream update into its own script

### PR DESCRIPTION
**Pull request by namespacebrian**

### CODE REVIEW

File: devops/scripts/deploy-d10-start-state.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# copy our script so it's available to run after we change branches
+cp devops/scripts/apply_drupal10_composer_changes.php /tmp
+
+# add the upstream repos as remotes - we need the 'public' upstream because d10 is based on it
+git remote add public "$UPSTREAM_REPO_REMOTE_URL"
+git fetch public
+
+git remote add drupal-10-start "$DRUPAL_10_REPO_REMOTE_URL"
+git fetch drupal-10-start
+
```
Comment (jazzsequence) [inline devops/scripts/deploy-d10-start-state.sh:24]: unnecessary extra line break

File: devops/scripts/deploy-d10-start-state.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# This script is pretty tailored to assuming it's running a fresh git clone.
+
+# Check github authentication; ignore status code 1 returned from this command
+ssh -T git@github.com
+
+# Fail fast on any future errors.
+set -euo pipefail
+
```
Comment (jazzsequence) [inline devops/scripts/deploy-d10-start-state.sh:10]: unnecessary extra line break

**Review by jazzsequence — APPROVED** [review]
there's a couple places where there are two carriage returns instead of one. other than that, this lgtm

**Comment by namespacebrian** [comment]
> there's a couple places where there are two carriage returns instead of one. other than that, this lgtm

Those were deliberate, but I can remove them.

**Pull request merged**